### PR TITLE
PETs duplicated after editing.

### PR DESCRIPTION
**Pull request by nicolocaruso85**
Updating a PET in a multilanguage site will duplicate it. I think we should define a default language (EN) for pets on installation and update existing templates.

screen shot 2017-02-13 at 10 07 50

### CODE REVIEW

**Review by salvoscala — COMMENTED** [review]
lgtm, wait on tests

**Comment by salvoscala** [comment]
@acrollet Can you CR this, thanks! :)

File: modules/roomify/roomify_dashboard/views/roomify_dashboard_handler_localized_edit_link.inc [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file
+ */
+
+/**
+ *
```
Comment (acrollet) [inline modules/roomify/roomify_dashboard/views/roomify_dashboard_handler_localized_edit_link.inc]: let's document what this class does

**Review by acrollet — APPROVED** [review]
CR passes after class is documented

**Pull request merged**